source/agora/consensus/SCPEnvelopeStore.d

```diff
@@ -181,12 +183,12 @@ unittest
 
     foreach (env; envelopes)
     {
-        envelope_store.add(env);
+        envelope_store.add(env, false);
     }
 
     assert(envelope_store.length == 2);
 
-    foreach (const ref SCPEnvelope env; envelope_store)
+    foreach (_, const ref SCPEnvelope env; envelope_store)
```

Comment:
Is the `_` used for indexing somewhere?

Reply:
Since there is only one `opApply` defined, we need to give 2 arguments to `foreach`. I didn't define a single argument `opApply` because, well, `opApply` is only used in two places, and one of them is this test.